File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { AlanPerlisQuotesSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = AlanPerlisQuotesSDK.test()
    equal(testsdk instanceof AlanPerlisQuotesSDK, true,
      'AlanPerlisQuotesSDK.test() must return a client synchronously')
  })

})

File: lua/test/exists_test.lua
-- AlanPerlisQuotes SDK exists test

local sdk = require("alan-perlis-quotes_sdk")

describe("AlanPerlisQuotesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
